src/config.ts

```diff
@@ -20,7 +25,7 @@ export const signInTextNeedsToBeReplaced = loginScreen?.changeSigninToLoginTextL
 
 // background color on login screen
 // used by app-web to load a colored background instead of lake image
-export const loginScreenBackgroundColor = loginScreen?.backgroundColor ?? null;
+export const showLoginScreenBackgroundImage = loginScreen?.showLoginScreenBackgroundImage ?? null;
```

Comment:
Searching this pull request, I can't see where showLoginScreenBackgroundImage is still used?

Reply:
done